strapi/strapi
# fix: Added correct object for plugin loading

### What does it do?

Set the correct uid for a plugin. 

### Why is it needed?

It used to output `undefined` in the `strapi` object for all the plugin uid's

### How to test it?

Log the content types for the plugins and verify the uid is correctly set now, instead of something like `plugin::users-permissions.undefined`. It should be `plugin::users-permissions.user`

### Related issue(s)/PR(s)

-
